docs/pages/platform/upgrading/2.2.mdx

```diff
@@ -0,0 +1,49 @@
+---
+meta:
+  title: "Upgrading to 2.2"
+  parentTitle: "Upgrading"
+  description: "Guide to upgrade to Liveblocks version 2.2"
+---
+
+We are making Thread "resolved" a first-class citizen property. You don't have
+to use thread's metadata to set a thread as resolved anymore.
+
+## All breaking changes are for Comments and only if you use a metadata "resolved"
+
+If you are using Comments without using a metadata `resolved`, there are no
```

Comment:
```suggestion
If you are using Comments without using the `resolved` metadata property, there are no
```

Reply:
Good suggestion, @CTNicholas. There are a couple more instances of this.

Reply:
Fixed!